resources/authgear/templates/en/web/authflowv2/setup_oob_otp.html

```diff
@@ -39,6 +38,18 @@ <h2 class="screen-description">
           {{ template "v2-setup-oob-otp-page-subtitle--email" }}
         {{ end }}
       </h2>
+
+      {{ if $.Error }}
+        {{ if not $display_otp_input_error }}
+          {{ template "authflowv2/__alert_message.html"
+            (dict
+              "Type" "error"
+              "Classname" "max-tablet:hidden mt-4"
+              "Message" (include "authflowv2/__error.html" .)
+            )
+          }}
+        {{ end }}
+      {{ end }}
```

Comment:
Where did you find this design?

btw, it looks like `__alert_message` should be something global in all screen, which displays any error not handled in that screen. Maybe we should create an issue to review the usage of `__alert_message` in other screens too?

Reply:
Desktop: https://www.figma.com/file/LRE8EfCaTfYxC0lk5AeMmE/New-Auth-UI?type=design&node-id=1148-41148&mode=design&t=PDG0RvzBvlSXJgDi-4
Mobile: https://www.figma.com/file/LRE8EfCaTfYxC0lk5AeMmE/New-Auth-UI?type=design&node-id=822-16106&mode=design&t=PDG0RvzBvlSXJgDi-4

You still need to tell `__alert_message` what error is ignored, which I think is screen-specific?

Reply:
Opened https://github.com/authgear/authgear-server/issues/3698